### ISSUE TITLE
tvOS 12 Space Scenes

### ISSUE THREAD

**Issue by Nyrad1** [cDima/Aerial#187]
I know theres an unofficial workaround but is there any timeline for a release of this screensaver with the new space scenes?

**Comment by Tiino-X83** [comment]
I agree with you, it would be impeccable to have the new wallpaper tvOS 12, as well as the option to choose with or without 4K / HDR
